parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -186,27 +186,14 @@ private Map<Long, Long> generateOffsetToStartPos(Schema schema) {
       return null;
     }
 
-    FileDecryptionProperties decryptionProperties =
-        (options == null) ? null : options.getDecryptionProperties();
-
-    ParquetReadOptions readOptions =
-        ParquetReadOptions.builder().withDecryption(decryptionProperties).build();
```

Comment:
These decryption properties should already be passed in via the normal options when doing the read to begin with. We already read the row group metadata in the ReadConf constructor. No need for the extra I/O as far as I can tell